modules/mono/editor/script_templates/VisualShaderNodeCustom/basic.cs

```diff
@@ -20,37 +21,37 @@ public override string _GetDescription()
         return "";
     }
 
-    public override long _GetReturnIconType()
+    public override Godot.VisualShaderNode.PortType _GetReturnIconType()
```

Comment:
```suggestion
    public override VisualShaderNode.PortType _GetReturnIconType()
```
The `_GetCode` method doesn't prefix `VisualShader` or `Shader` with `Godot`, so I don't think that's necessary here either